dht/token-sharding.hh

```diff
@@ -28,6 +28,10 @@ struct shard_and_token {
     token token;
 };
 
+/// Represents a set of shards that own a given token on a single host.
+/// It can be either of: none, single shard (no tablet migration), or two shards (during intra-node tablet migration).
+using shard_replica_set = utils::small_vector<unsigned, 2>;
```

Comment:
boost::static_vector would be a better choice, as much as I hate recommending boost.

Reply:
Yes